layer/flavors/base.py

```diff
@@ -1,12 +1,21 @@
 import inspect
 from abc import ABCMeta, abstractmethod, abstractproperty
+from dataclasses import dataclass
 from pathlib import Path
+from typing import Callable, Optional
 
+import pandas as pd
 from layerapi.api.entity.model_version_pb2 import ModelVersion
 
 from layer.types import ModelArtifact
 
 
+@dataclass(frozen=True)
+class ModelRuntimeObjects:
+    model_artifact: ModelArtifact
```

Comment:
maybe rename to model_object ? and rename the type as well

Reply:
I've renamed it.